### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by dasarpan007**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

### CODE REVIEW

**Comment by ldewsnap** [comment]
Feasibility:
As I understand it, both of the programs described in the proposal are established, functional, and are accessible, so the project described here does seem feasible. On the other hand, it wasn't clear to me whether these programs are something you've already used for other work or if you're new to them.

Scope:
It was not clear to me how large/finely incremented the grid of models will be, nor its actual structure (i.e. is it Ldot varying on one axis of the grid, and inclination on another? Is viscosity being left set to 1 as is mentioned to "often" be the case, or is that parameter also being varied?). It's also not clear to me how substantial one of these models is, such as how much processing power and time they take. However, the general principle seems reasonable in scope.

Justification:
The section on scientific relevance was somewhat hard to follow, especially because the wording was fragmented. I think I get the general idea (data can be compared against eh resulting grid of models in order to see which fits each Be star best) but it could have been clearer.

**Comment by Virachan** [comment]
Hi Jason,

Please see my comments below.

Scope:
Although the project plan is systematically outlined, the scope of the project is not clear.  It seems like the project is divided into two parts -- first, focussing on modelling the Be star disc and the second, deriving the observables from the developed model.  However, the specifics in each of those parts are not explicitly stated.  For example, what considerations are made in a standard grid of models.  It is not clear how many models will be used and how large they would be.  Also, how extensive is the use of the HDUST software package that will derive the observables such as the Halpha line is not clearly described.  This makes it difficult to infer how large or small amount of work will be required to obtain the desired results.

Feasibility:
The modelling equations are clearly stated that will be used in the first part of the project.  As such, solving the disc evolution equation would not require special programs or libraries.  However, the amount of computing power that is needed might be large.  It is not clear if the models will be run on a remote server or if the local system that will be used would be sufficient to achieve the modelling results.

Justification:
The motivation behind modelling Be star discs and deriving observable parameters to understand the evolution of the disc in these objects is reasonably justified.  Modelling Be star discs and observables therein to study the mass loss in high-mass stars is a challenging problem -- a solution to which can provide useful insights into modelling disc structures in other compact objects (e.g., black hole accretion disks).

Hope this feedback is helpful!
